### ISSUE TITLE
[TBC Bloodmyst Isle] Missing Quest Indicator ( ! )

### ISSUE THREAD

**Issue by Vyperhand** [chromiecraft/chromiecraft#5504]
### What client do you play on?

enUS

### Faction

Alliance

### Content Phase:

1-19

### Current Behaviour

Breaking this out into its own entry per request.

Techician Zhanaa does not display the ! icon for her final quest, Travel to Azure Watch. (https://www.wowhead.com/quest=9313/travel-to-azure-watch)

UPDATE - On retest I noticed that after turning in the previous quest, the ! icon displays for a fraction of a second before the scene starts and never reappears once it has completed. This might be an issue/conflict with whatever is auto-accepting the 1-5 zone quests, in that case.  I was curious enough I created a third character and verified the behavior, as did my wife on her account.  This quest is the zone-change breadcrumb, so if we get any players who haven't played this zone before, that could be problematic.

The low-level quests toggle does not seem to make it visible again.

### Expected Blizzlike Behaviour

The character should display her ! again showing that she has a quest to offer once the in-game cutscene completes.

### Source

Default questgiver behavior is to display ! unless the quest would be grey.

### Steps to reproduce the problem

1. Set a character to have completed all the starter quests in the Draenei starter zone, with The Emitter ready to turn in.
2. Turn the quest in, you may see a ! for a brief moment before the cutscene begins.  This happened on 2/4 tests.
3. After the cutscene, the ! does not reappear, but the NPC has the speech bubble interact icon.
4. Interacting with the NPC now properly offers the Travel to Azure Watch breadcrumb quest.

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/ce6ca597b54e64f328cacddae5f003cdc41a343d

### Operating system

Windows 10 Home 21H2

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-bg-item-reward](https://github.com/azerothcore/mod-bg-item-reward)
- [mod-BreakingNewsOverride](https://github.com/AnchyDev/BreakingNewsOverride)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-cta-switch](https://github.com/azerothcore/mod-cta-switch)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna](https://github.com/azerothcore/mod-eluna)
- [mod-instanced-worldbosses](https://github.com/nyeriah/mod-instanced-worldbosses)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-low-level-rbg](https://github.com/azerothcore/mod-low-level-rbg)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-progression-system](https://github.com/azerothcore/mod-progression-system)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-rdf-expansion](https://github.com/azerothcore/mod-rdf-expansion)
- [mod-transmog](https://github.com/azerothcore/mod-transmog)
- [mod-weekend-xp](https://github.com/azerothcore/mod-weekend-xp)
- [mod-zone-difficulty](https://github.com/azerothcore/mod-zone-difficulty)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-event-scripts](https://github.com/55Honey/Acore_eventScripts)
- [lua-exchange-npc](https://github.com/55Honey/Acore_ExchangeNpc)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

**Comment by Vyperhand** [comment]
Still an issue as of this morning.

![WoWScrnShot_062423_100031](https://github.com/chromiecraft/chromiecraft/assets/131016474/05dd7512-9d58-4f15-b262-16fd693c3cba)

**Comment by Vyperhand** [comment]
The quest can still be taken from the NPC.  The ! still flickers very briefly before the cutscene starts and never reappears.

**Comment by Rorschach91** [comment]
I tested the quest and I haven't found this bug. 
Sometimes quest indicators can be bugged by an addon or a broken client.
If you want to be sure, try to relog when this bug appears or use another client.

Thank you for your report anyway.

**Comment by Vyperhand** [comment]
It was a conflict with the auto-completion module that was being used for the starter areas.  It's possible that got addressed a few months back.
    On Monday, August 12, 2024 at 07:52:21 AM PDT, Rorschach91 ***@***.***> wrote:   

 


I tested the quest and I haven't found this bug.
Sometimes quest indicators can be bugged by an addon or a broken client.
If you want to be sure, try to relog when this bug appears or use another client.

Thank you for your report anyway.
